yt_dlp/extractor/vk.py

```diff
@@ -25,6 +25,7 @@
     url_or_none,
     urlencode_postdata,
     urljoin,
+    traverse_obj,
```

Comment:
sort this

Reply:
let it be. I am planning to add isort to CI soon and will clean up all extractors when I do.